wcfsetup/install/files/lib/util/HeaderUtil.class.php

```diff
@@ -94,6 +94,11 @@ public static function sendHeaders()
             @\header('X-Frame-Options: SAMEORIGIN');
         }
 
+        // send Permissions-Policy
+        if (HTTP_DISABLE_FLOC) {
+            @\header('Permissions-Policy: interest-cohort=()');
```

Comment:
```suggestion
            @\header('permissions-policy: interest-cohort=()');
```
We use lowercase names for all newly added headers.